chapters/en/slides/module2/module2_04.Rmd

```diff
@@ -0,0 +1,143 @@
+---
+params:
+  dynamictitle: "module2_04"
+title: "`r params$dynamictitle`"
+output:
+  md_document:
+    variant: gfm
+    pandoc_args: "--markdown-headings=atx"
+---
+
+```{r setup, include=FALSE}
+source('../../../../slide-setup.R')
+# Remember to also manually update the YAML title above
+```
+
+type: slides
+
+# Creating subplots via faceting
+
```

Comment:
Can you add a slide before this to set this up? Just a reminder of the question, we asked and answered with this histogram, without colouring by region. And then we can formulate a new question that would be answered by what is done on this slide here.